source/stylesheets/base/_layout.scss

```diff
@@ -11,3 +11,11 @@ a {
 	}
 }
 
+.layout {
+	max-width: 60em;
+	@include margin($spacing-base);
+
+    @include breakpoint($breakpoint-medium) {
+		@include margin($spacing-large auto);
+    }
+}
```

Comment:
It’d be great to get consistent indentation throughout the styles and views. I’d specifically recommend using spaces instead of tabs. I’m a two-space fan, but maybe you like four better.

```scss
.layout {
  @include margin($spacing-base);
  max-width: 60em;

  @include breakpoint($breakpoint-medium) {
    @include margin($spacing-large auto);
  }
}
```

Reply:
Yeah I've gotten this feedback before - I just enabled whitespace in my IDE so I'll be able to line these up better - I'm a fan of 2 spaces as well